awx/api/serializers.py

```diff
@@ -1261,6 +1261,7 @@ def get_related(self, obj):
             dict(
                 tokens=self.reverse('api:o_auth2_application_token_list', kwargs={'pk': obj.pk}),
                 activity_stream=self.reverse('api:o_auth2_application_activity_stream_list', kwargs={'pk': obj.pk}),
+                organization=self.reverse('api:organization_detail', kwargs={'pk': obj.organization.pk}),
```

Comment:
This is a nullable field, so you need to check if it exists first then add it to the dict.  Also, I'd recommend using `obj.organization_id` to avoid the potential query to get the organization object.

Reply:
Fixed.